bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -144,7 +162,21 @@ class ProjectBuilder {
             '// GENERATED FILE - DO NOT EDIT\n' +
             'include ":"\n' + settingsGradlePaths.join(''));
 
-        // Update dependencies within build.gradle.
+        // Update SDK versions within root build.gradle - quick fix:
+        let rootGradle = fs.readFileSync(path.join(this.root, 'build.gradle'), 'utf8');
+        rootGradle = rootGradle.replace(
+            /(defaultMinSdkVersion)\s*=\s*\d+\s*\/\//,
+            '$1=' + androidManifestInfo.minSdkVersion + ' //');
+        rootGradle = rootGradle.replace(
+            /(defaultTargetSdkVersion)\s*=\s*\d+\s*\/\//,
+            '$1=' + androidManifestInfo.targetSdkVersion + ' //');
+        rootGradle = rootGradle.replace(
+            /(defaultCompileSdkVersion)\s*=\s*\d+\s*\/\//,
+            '$1=' + androidManifestInfo.targetSdkVersion + ' //');
```

Comment:
Just to confirm, we're using the `targetSdkVersion` as the `defaultCompileSdkVersion`, right?

Reply:
Comment added